src/HealthcareApis/HealthcareApis/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Removed population of access policies
```

Comment:
Maybe say something like access policies are no longer defaulted to the current principal?